src/_posts/gamestate.md

```diff
@@ -0,0 +1,346 @@
+---
+title: "Managing Application and Game State in A-Frame"
```

Comment:
I don't have a good title off the top of my head, but perhaps tweak the title to say something about how maintaining game state lends itself to reliability, testability, maintenance, resilient to common errors, etc.?